### PR TITLE
[Playwright] PEPPER-645 Update project dependency to latest versions

### PR DESCRIPTION
**Pull request by aweng98**
Update all dependencies to latest versions. Verify all tests still run okay on localhost and docker container.

Screenshot 2023-02-28 at 10 11 19 AM

Passed in CircleCI [run](https://app.circleci.com/pipelines/github/broadinstitute/ddp-angular/23694/workflows/f15b20f7-d4ef-4668-ac49-c51d55c110a4).

I had to update few screenshots due to new images are a little different. One known test failure unrelated to upgrade is `tests/pancan/enrollment/adult-self-and-child-enrollment.spec.ts`

### CODE REVIEW

File: playwright-e2e/pages/page-base.ts [hunk 1]
```diff
@@ -241,21 +242,16 @@ export default abstract class PageBase implements PageInterface {
...
     const { state } = opts;
 
-    const [selected] = await this.country().select().selectOption(country);
-    /*
-    // assert in V1.29. Uncomment after upgrade Playwright
-    await expect(async () => {
-      const selectedOption = await this.country().select().evaluate<string, HTMLSelectElement>((node) => node.value);
-      expect(selectedOption).toEqual(country);
-    }).toPass();
-    */
```
Comment (aweng98) [inline playwright-e2e/pages/page-base.ts:254]: Created new `assertSelectedOption` function for asserting selected option in `Select` element. `.toPass()` is one of many new features in new Playwright version.

File: playwright-e2e/playwright.config.ts [hunk 2]
```diff
@@ -75,7 +75,10 @@ const testConfig: PlaywrightTestConfig = {
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: process.env.CI ? 'on-first-retry' : 'retain-on-failure',
-    screenshot: 'only-on-failure',
+    screenshot: {
+      mode: 'only-on-failure',
+      fullPage: true
+    },
```
Comment (aweng98) [inline playwright-e2e/playwright.config.ts:81]: `fullPage: true` is a new feature.

File: playwright-e2e/playwright.config.ts [hunk 3]
```diff
@@ -16,8 +16,8 @@ const testConfig: PlaywrightTestConfig = {
   globalSetup: require.resolve('./fixtures/global-setup'),
   testDir: '.',
   testMatch: '**/*.spec.ts',
-  /* Maximum timeout per test. Test should be short and takes less than 2 min or 3 min in docker container to run */
-  timeout: process.env.CI ? 180 * 1000 : 120 * 1000,
+  /* Maximum timeout per test. Each test should be short and takes less than 3 min to run */
+  timeout: 180 * 1000,
```
Comment (aweng98) [inline playwright-e2e/playwright.config.ts:20]: Set test max timeout same for CI and localhost, to 3 min. During my local testing, one test ran over 2 min and it failed due to timeout.

File: playwright-e2e/utils/test-utils.ts [hunk 4]
```diff
@@ -4,7 +4,6 @@ import Checkbox from 'lib/widget/checkbox';
 import Radiobutton from 'lib/widget/radiobutton';
 import Select from 'lib/widget/select';
 import axios from 'axios';
-import { Response } from 'playwright-core';
```
Comment (aweng98) [inline playwright-e2e/utils/test-utils.ts:7]: Encountered an issue with build in CircleCI (see screenshot). Removal has fixed issue.

Screenshot 2023-02-28 at 3 35 11 PM

**Review by GiCharkviani — APPROVED** [review]
LGTM

**Pull request merged**